src/layouts/components/Editor/Iframe.ts

```diff
@@ -0,0 +1,118 @@
+import { Node, nodeInputRule, nodePasteRule } from "@tiptap/core"
+import { ReactNodeViewRenderer } from "@tiptap/react"
+import _ from "lodash"
+
+import { LegacyBlockView } from "./LegacyBlockView"
+
+export interface IframeOptions {
+  allowFullscreen: boolean
+  HTMLAttributes: {
+    [key: string]: any
+  }
+}
+
+declare module "@tiptap/core" {
+  interface Commands<ReturnType> {
+    iframe: {
+      /**
+       * Add an iframe
+       */
+      setIframe: (options: { src: string }) => ReturnType
+    }
+  }
+}
```

Comment:
required so that other files can access the command like so: `editor.commands.iframe.setIframe`

Reply:
rather than redeclaring this here, could we instead add a thin wrapper around this and call this `IsomerTipTapCore`? Now devs have to remember to export from here rather than the default tiptap core that is exposed by the library.

Reply:
not sure i'm understanding you here. the point of this declaration here is for [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) so that the default editor used by the library has the typing on it.